src/view/Input.js

```diff
@@ -0,0 +1,39 @@
+import { Console } from "@woowacourse/mission-utils";
+
+class Input {
+  static #CAR_NAME_INPUT_MESSAGE =
+    "경주할 자동차들의 이름을 쉼표(,)로 구분해서 입력하세요.\n";
+
+  static #MOVE_COUNT_INPUT_MESSAGE =
+    "이동을 시도할 횟수를 숫자로 입력해주세요.\n";
+
+  /**
+   * @async
+   * @param {string} message - 입력 전에 출력할 메시지
+   * @returns {Promise<string>} - 입력받은 문자열
+   */
+  static async #readLine(message) {
+    return Console.readLineAsync(message);
+  }
+
+  /**
+   * 자동차 이름들을 입력받는 메서드
+   * @returns {Promise<string>} - 입력받은 자동차 이름들 (쉼표로 구분)
+   */
+  static async readCarNames() {
+    const carNames = await Input.#readLine(Input.#CAR_NAME_INPUT_MESSAGE);
+    return carNames;
+  }
```

Comment:
```suggestion
static async readCarNames() {
    const carNames = await Console.readLineAsync(message);
    return carNames;
  }
```
굳이 두 개의 매서드로 분리할 필요가 있을까요?🤔

Reply:
어디까지 추상화 해야 하는지에 대한 결정은 저도 고민을 많이 했던 부분인데요. 🤔 여기서 `Console.readLineAsync` 을 추상화한 메서드를 따로 만든 이유는 이 부분이 외부 라이브러리를 사용했기 때문입니다. 외부 라이브러리를 사용하는 만큼 외부 라이브러리 변경 사항 등이 제가 짠 코드에 미치는 영향을 최소화해주고 싶어서 추상화 메소드로 한번 더 감싸주는 방법을 써 봤어요 ㅎㅎ